src/components/LoginPage/SideWave.tsx

```diff
@@ -1,26 +1,25 @@
 import * as S from './style';
 import { useRecoilState } from 'recoil';
 import { ViewWidth } from '../../Atom/Atoms';
-import { useEffect, useRef } from 'react';
+import React, { useEffect, useRef } from 'react';
```

Comment:
```suggestion
import { useEffect, useRef } from 'react';
```
다른 곳에서는 안 붙였으니깐 이건 필요 없을 듯